packages/KEmptyState/KEmptyState.vue

```diff
@@ -1,10 +1,23 @@
 <template>
   <div class="empty-state-wrapper">
     <div class="empty-state-title">
-      <h5><slot name="title"/></h5>
+      <div
+        v-if="isError"
+        class="warning-icon card-icon mb-4"
+      >
+        <KIcon
```

Comment:
I'm a little worried our design has be degraded due to using a lower fidelity kicon instead of the warning svg we currently use in KM. @steph-kalacis which one do you prefer (keeping in mind that the one on the right is what we use for a kicon and use it in alerts):
![image](https://user-images.githubusercontent.com/5770711/75556526-65aa7a00-5a0c-11ea-8eb9-f7dba58cf5ab.png)

Reply:
we could also potentially change the KIcon SVG to the KM warning one

Reply:
I prefer the option to the left! Thanks for asking